src/types/types.ts

```diff
@@ -158,3 +158,38 @@ export interface ICreateCandidateFinalState {
   comment: string;
   submissionDate: string;
 }
+
+export interface ICreateCvInitialState {
```

Comment:
CreateCvRequest this should be placed in a separate type file what could be named api.ts (folder is correct)